sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/models/FileHttpHeaders.java

```diff
@@ -69,9 +69,9 @@ public String getFileContentType() {
      * file. The default type is 'application/octet-stream'.
      *
      * @param fileContentType the fileContentType value to set.
-     * @return the FileHTTPHeaders object itself.
+     * @return the FileHttpHeaders object itself.
      */
-    public FileHTTPHeaders setFileContentType(String fileContentType) {
+    public FileHttpHeaders setFileContentType(String fileContentType) {
```

Comment:
Based on API review feedback, there is a [discussion](https://github.com/Azure/azure-sdk-for-java/issues/5677#issuecomment-540785823) to remove the word `file` from setter and getters in this class.

Reply:
Will not be done in this PR